packages/http/fetch/package.json

```diff
@@ -40,6 +40,7 @@
 	"dependencies": {
 		"@microsoft/kiota-abstractions": "^1.0.0-preview.22",
 		"@opentelemetry/api": "^1.2.0",
+		"guid-typescript": "^1.0.9",
```

Comment:
Out of curiosity, won't this be added as an indirect dependency from the abstractions package. If so, do we need to add it here? and why not add to the other packages as well(serialization-form, serialization-json)?

Reply:
Yes you're right ill remove it

Reply:
actually in typescript/javascript land, if your code has a direct dependency on a package (i.e. you have an import pointing to that package in your code), it should be declared as a dependency. Granted we control abstractions as well, but I learnt that the hard way...

Reply:
@baywet does it means the documentation will have to be updated once we merge because the generated code, in certain cases, now contains `import { Guid } from "guid-typescript";`